api/nodes/views.py

```diff
@@ -646,7 +646,7 @@ def get_serializer_class(self):
 
     # overrides ListBulkCreateJSONAPIView, BulkUpdateJSONAPIView
     def get_queryset(self):
-        queryset = self.get_queryset_from_request()
+        queryset = self.get_default_queryset()
```

Comment:
I'm not sure this is the correct thing to do. `get_queryset_from_request` respects query parameters (e.g. filtering); `get_default_queryset` does not. This is making the tests (correctly) fail.